src/components/app.jsx

```diff
@@ -1,28 +1,97 @@
 import React from 'react';
+// import PropTypes from 'prop-types';
 import LocationDetails from './location-details';
-import PropTypes from 'prop-types';
+import ForecastSummaries from './forecast-summaries';
+import '../styles/app.scss';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
 
-const App = props  => (<LocationDetails
+const URL_DATA = 'https://mcr-codes-weather.herokuapp.com/forecast';
 
-  city={props.location.city}
-  country={props.location.country}
-/>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
 
-App.propTypes = {
+    this.state = {
+      // put error in state- ?google
+      searchError: '',
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        contry: '',
+      },
+
+    };
+    this.handleForecastSelect = this.handleForecastSelect.bind(this);
+    this.handleLocationSearch = this.handleLocationSearch.bind(this);
+  }
+
```

Comment:
Purely a stylistic issue, but try and have your lifecycle methods listed first, then your helper functions, then handlers and finally you render method.